base/workflow/R/do_conversions.R

```diff
@@ -9,6 +9,10 @@
 ##' @author Ryan Kelly, Rob Kooper, Betsy Cowdery, Istem Fer
 
 do_conversions <- function(settings, overwrite.met = FALSE, overwrite.fia = FALSE, overwrite.ic = FALSE) {
+
+  # extract site info from settings to be utilizes later in processing.
+  site.info <- settings$run$site
+
```

Comment:
Better to use it directly from inside the existing `settings` object

Reply:
I'd been experimenting to see if we have any instance which can be directly utilized with `site` rather than querying it. Although now I've shifted it to a new function, it's nice to use it from within. Thanks for suggestion